assets/css/v2/style.css

```
@@ -508,6 +508,33 @@ atomic-search-layout atomic-layout-section[section="search"] {
   box-shadow: 3px 3px 0px var(--color-shadow);
 }
 
+@keyframes rotateAnimationToCloseIcon {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(180deg);
+  }
+}
+
+@keyframes rotateAnimationToOpenIcon {
+  0% {
+    transform: rotate(180deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
+}
+
+button:has(~ .product-selector[style*="block"])
+  > .product-selector-button-icon {
+  animation: rotateAnimationToCloseIcon 0.25s ease-in-out forwards;
+}
+
+button:has(~ .product-selector[style*="none"]) > .product-selector-button-icon {
+  animation: rotateAnimationToOpenIcon 0.25s ease-in-out forwards;
+}
+
```

You can use native transform here, instead of setting up keyframes.
Changed the transform to `90deg` as well, and set that to the default so the chevron points down initially.
Seeing it in action, I'm not sure if the animation works. Especially since the product selector itself appears before the chevron has turned. 🤔 
Will run it by @danielledeleo as well and see :) .
 
```suggestion
.product-selector-button-icon {
  transform: rotate(90deg);
}

button:has(~ .product-selector[style*="block"])
  > .product-selector-button-icon {
  transition: transform 0.25s ease-in-out;
  transform: rotate(0deg);
}

button:has(~ .product-selector[style*="none"]) > .product-selector-button-icon {
  transition: transform 0.25s ease-in-out;
  transform: rotate(90deg);
}
```

---

Yeah, remove the animations :D